charts/base-cluster/templates/weave/_helpers.tpl

```diff
@@ -0,0 +1,3 @@
+{{- define "weaveworks.password" -}}
+{{ include "common.secrets.passwords.manage" (dict "secret" "cluster-user-auth" "key" "password" "providedValues" (list "weaveworks.adminUser.password") "length" 20 "strong" false "skipB64enc" true "skipQuote" true "context" $) | bcrypt }}
```

Comment:
Do we *really* need bcrypt? Then you have to totally revamp this, as bcypt is random every time, see https://gitlab.teuto.net/4teuto/dev/teuto-portal/teuto-portal/-/blob/main/charts/teuto-portal/templates/mailpit/auth-secret.yaml

We can also talk about this, as it's kinda complicated

Reply:
unfortunately, yes. The dashboard expects a bcrypted input and there is no other option to deploy a password for the user.

Reply:
updated